.env

```
@@ -0,0 +1,9 @@
+AI_API_KEY=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
+AI_API_URL=https://openrouter.ai/api/v1/chat/completions
+AI_MODEL=arcee-ai/trinity-large-preview:free
+CURRENCY_BASE_URL=https://api.currencybeacon.com/v1
+CURRENCY_TOKEN=mvhREoY2f8UjpJ45vjT7RVn7JSvihea4
+ADMOB_BANNER_ANDROID=ca-app-pub-4574158711047577/3296197854
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ADMOB_BANNER_ANDROID key should go before the AI_API_KEY key

(UnorderedKey)